### DISCUSSION TITLE
Images are not being downloaded

### DISCUSSION THREAD

**Discussion by GonzaloAndrade** [vasani-arpit/WBOT#295] [General]
Tested with the latest version in windows 10
![help](https://user-images.githubusercontent.com/1771862/149673589-77dc6177-f68b-4ac2-bb28-0edb71676369.png)

Got any ideas?

**Comment by vasani-arpit** [comment]
WA may have changed few things. Will check and get back to you.

**Comment by vasani-arpit** [comment]
I can confirm that this is an issue. I have a solution in mind will add it in few days. will probably use @open-wa/wa-decrypt to download and decrypt the media.

**Comment by vasani-arpit** [comment]
Resolved in latest [version](https://github.com/vasani-arpit/WBOT/releases).
